database/pgx/pgx.go

```diff
@@ -121,6 +121,11 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 }
 
 func (p *Postgres) Open(url string) (database.Driver, error) {
+	// Driver is registered as pgx, but connection string must use postgres schema
+	// when making actual connection
+	// i.e. pgx://user:password@host:port/db => postgres://user:password@host:port/db
+	url = strings.Replace(url, "pgx://", "postgres://", 1)
```

Comment:
Avoid manipulating URLs via string operations. Instead, set `purl.Scheme`.
I like the documentation in the comments though. Let's keep that!